firefox_ui_tests/functional/private_browsing/test_about_private_browsing.py

```diff
@@ -2,65 +2,54 @@
 # License, v. 2.0. If a copy of the MPL was not distributed with this
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
 
-from marionette_driver import By
+from marionette_driver import By, Wait
 
 from firefox_ui_harness.decorators import skip_if_e10s
 from firefox_ui_harness import FirefoxTestCase
 
+from firefox_puppeteer.ui.windows import BrowserWindow
+
 
 class TestAboutPrivateBrowsing(FirefoxTestCase):
 
     def setUp(self):
         FirefoxTestCase.setUp(self)
-        self.pb_url = self.marionette.absolute_url('private_browsing/about.html?')
 
-    def tearDown(self):
-        self.marionette.close()
+        self.support_url = 'about:blank?'
+        self.pb_url = self.support_url + 'private-browsing'
```

Comment:
this seems overkill to have support_url and pb_url where as both are only used once.  I would rather see no variables here, but one would be fine.

Can you also comment why we need to set the preference to this value on teardown?  It isn't clear to me by reading the code